### DISCUSSION TITLE
How to do manual focusing?

### DISCUSSION THREAD

**Discussion by stevempotter** [feelfreelinux/octo4a#152] [Q&A]
I saw on the Telegram discussion that since v. 1.0.0 there is the ability to disable autofocus. I cannot find that. Where is it? How do you do manual focus? I am using a Huawei P20 Pro. My videos are ruined by the focus changing a lot due to autofocusing.

**Comment by stevempotter** [comment]
OK, I found the Disable Autofocus switch at the very bottom of the Settings for Octoprint on the phone (but nowhere in the settings on the Octoprint server remote UI). Now the camera is just always out of focus. How do I focus it?

**Reply by cjmorri1018** [reply]
Did you figure this out?
